en/docs/reference/connectors/facebook-ads-connector/facebook-ads-configuration.md

```diff
@@ -725,6 +725,154 @@ To use the Facebook Ads connector, first create the connection with your configu
         "ad_id": "456456456"
     }
     ```
+??? note "createCustomAudience"
+    The `createCustomAudience` operation creates a custom audience.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td>adAccountId</td>
```

Comment:
Please [format](https://github.com/wso2/docs-mi/pull/1013/files#diff-081223524980be6997832380365714049f5478394a5b67f494b265a323d0d716R18) the values of all the property names inside `<code>` tags. Update all the occurrences.